vent/io/devices/valves.py

```diff
@@ -29,7 +29,7 @@ def form(self, form):
         """ Performs validation on requested form and then sets it.
 
         Args:
-            f (str): The form of the solenoid; can be either `Normally Open` or `Normally Closed`
+            form (str): The form of the solenoid; can be either `Normally Open` or `Normally Closed`
         """
         if form not in self._FORMS.keys():
             raise ValueError('form must be either NC for Normally Closed or NO for Normally Open')

```

Comment:
error no longer reflects check, should have error be programmatic, `f'must be one of {self._FORMS.keys()}'`